umh.docs.umh.app/content/en/docs/datacontracts/_index.md

```diff
@@ -0,0 +1,213 @@
+
+#### Schema dependent content
+
+Depending on the schema used, the next parts of the topic may differ. For
+example, in the `_historian' schema, you can either attach your payload
+directly or continue to group tags. See the Historian Data Contract page
+for more details.
+
+#### Allowed Characters
+
+Topics can consist of any letters (`a-z`, `A-Z`), numbers (`0-9`), and the
+symbols (`-` & `_`). Be careful to avoid `.`, `+`, `#`, or `/` as these are
+special symbols in Kafka or MQTT.
+
+Note that our topics are case-sensitive, so `umh.v1.ACMEIncorporated` is
+**not** the same as `umh.v1.acmeincorporated`.
+
+### Payload structure
+
+A Data Contract can include payload rules. For example, in the Historian Data
+Contract, you must include a timestamp in milliseconds and a key-value pair.
+
+These requirements are unique to each data contract.
+
+### Data Flow Components
+
+Data Flow Components can be of three different types: Protocol Converter, Data
+Bridge, or Custom Data Flow Component. All are based on
+[BenthosUMH](https://github.com/united-manufacturing-hub/benthos-umh).
+
+#### Protocol Converter
+
+You have probably already created a Protocol Converter and are familiar with
+its purpose. You use them to get data from different sources into your
+instances. You format the data into the correct payload structure and send it
+to the correct topics.
+
+#### Data Bridges
+
+Data Bridges are placed between the two message brokers in the Unified
+Namespace, Kafka and MQTT, and allow messages to be passed between them.
+The default Data Bridges are the two between MQTT and Kafka for the `_historian'
+schema. Each Data Bridge is unidirectional and specific to one schema.
+
+#### Custom Data Flow Components
+
+To meet everyone's needs and enable stream processing, you can add Custom Data
+Flow Components (creative naming is our passion). Unlike Protocol Converters or
+ata Bridges, you have full control over their configuration, which makes them
+incredibly versatile, but also complicated to set up. As a result, they must be
+manually enabled by switching to Advanced Mode in the Management Console Settings.
+
+### Other Data Contracts
+
+Data Contracts can build on existing contracts. For example, if you want to
+calculate KPIs, you can send the raw data to `_historian`, process it with a
+Custom Data Flow Component, and publish it to a new schema. The new Data
+Contract uses the Historian to collect data from the machines and store it in
+the database.
+
+## Summary Historian
+
+As explained above, the Historian Data Contract is the only standard Data
+Contract in the UMH. Its purpose is to ensure that the data to be stored in the
+database follows the correct format. For all information about the specifics of
+this Data Contract, see the
+[Historian Data Contract](https://umh.docs.umh.app/docs/datacontracts/historian/)
+page.
+
+## Summary Custom Data Contract
+
+Since we can never fully address every use case of the UMH, you can also create
```

Comment:
why are there two summaries here?

Reply:
to give an overview about the next pages.
But they are a bit redundant, given that the pages are listed below...
I will remove them.